autopush/router/apnsrouter.py

```diff
@@ -146,18 +147,22 @@ def _route(self, notification, router_data):
         try:
             apns_client.send(router_token=router_token, payload=payload,
                              apns_id=apns_id)
-        except ConnectionError:
+        except ConnectionError as ex:
             self.ap_settings.metrics.increment(
                 "updates.client.bridge.apns.connection_err",
                 self._base_tags
             )
+            self.log.error("Connection Error sending to APNS",
```

Comment:
pjenvey and I discussed that initially in channel. Not sure that this qualifies as a failure, since it's not killing the process, just invoking an error for a transaction.